settings.gradle.kts

```diff
@@ -3,6 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/*
+ * Copyright 2024 Broadcom. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Should this not replace the existing header?

Reply:
Fixed Intellij idea Copyright headers